.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -591,7 +591,14 @@ sqs_Scenario_SendReceiveBatch:
               snippet_tags:
                 - python.example_code.sqs.Scenario_SendReceiveBatch
   services:
-    sqs: {CreateQueue, DeleteQueue, SendMessageBatch, ReceiveMessage, DeleteMessageBatch}
+    sqs:

```

Comment:
Is this going to be an issue, @Laren-AWS ?

Reply:
I suppose if the linter allows it, I'll allow it. 😄
It should be fine in the build.

Reply:
Prettier did something funky, sorry :/